id-authentication-default.properties

```diff
@@ -271,6 +271,7 @@ ida-topic-fraud-analysis=IDA_FRAUD_ANALYTICS
 ida-topic-auth-anonymous-profile=ANONYMOUS_PROFILE
 ida-topic-pmp-oidc-client-created=OIDC_CLIENT_CREATED
 ida-topic-pmp-oidc-client-updated=OIDC_CLIENT_UPDATED
+ida-topic-remove-id-status=REMOVE_ID_STATUS
```

Comment:
Where is the websub secret configured for this topic?

Reply:
secret is not required in case of event publishing to websub, it needs only authentication.